bundles/org.openhab.binding.clementineremote/README.md

```diff
@@ -0,0 +1,334 @@
+# ClementineRemote Binding
+
+<img style="float: right; margin: 0 0 5px 5px" src="doc/screenshot.png" alt="Screenshot" />
+This binding shall bring the benefits of over-the-network control of the [Clementine-Player] to OpenHab!
+
+It allows control of playback, position and volume and allows display of various title information:
+
+* Artist
+* Album
+* Title
+* current position within track
+* Album cover
+
+
+[→ useful Links][links]
+
+## Discovery
+
+This thing does (currently) not provide auto-discovery.
+You will have to manually configure clementine to allow remote connections.
```

Comment:
Is there a howto link? I don't know this player, maybe it is super obvious and not needed, but you can think of adding a line or two where this setting in the player is?

Reply:
Good suggestion. I added a link to the documentation in the Clementine Android Remote wiki.